CMakeLists.txt

```diff
@@ -231,6 +231,36 @@ set(TARGET_LIBDFT "sleefdft")
 # Check subdirectories
 add_subdirectory("src")
 
+# Install the CMake package config
+include(CMakePackageConfigHelpers)
+
+write_basic_package_version_file(
+    sleefConfigVersion.cmake
+    COMPATIBILITY SameMajorVersion
```

Comment:
Is this the intended versioning strategy of sleef?

Reply:
Actually there is no strict rule for compatibility.

Reply:
Would you commit to major version level compatibility? You technically already do with using the project major version for SOVERSION.

Reply:
I think that's fine.
Compatibility rules are not strictly kept.
There is only a loose rule to keep backward compatibility for functions that are used in other projects.
For functions that do not seem to be used in other projects, compatibility is sometimes broken.